### PR TITLE
Added support for Anthropic's Claude LLM as a backend for the ai tool

### PR DESCRIPTION
**Pull request by oblivia-simplex**
The user can now use Anthropic's LLM, Claude, as an alternative to OpenAI's models, if they have an Anthropic API key.

### CODE REVIEW

**Comment by disconnect3d** [comment]
Please fix the lint issues: https://github.com/pwndbg/pwndbg/actions/runs/4632075092/jobs/8195702323?pr=1661

> pwndbg/commands/ai.py:12:1: F401 'textwrap' imported but unused
> import textwrap

File: pwndbg/commands/ai.py [hunk 1]
```diff
@@ -414,8 +465,9 @@ def ai(question, model, temperature, max_tokens, verbose, list_models=False, com
         last_question.pop(0)
         last_answer.pop(0)
 
-    term_width = os.get_terminal_size().columns
-    answer = textwrap.fill(res, term_width, replace_whitespace=False)
+    # term_width = os.get_terminal_size().columns
+    # answer = textwrap.fill(res, term_width, replace_whitespace=False)
+    answer = res
     print(M.success(answer))
```
Comment (disconnect3d) [inline pwndbg/commands/ai.py]: Should we just print `res` and remove the comments?

File: pwndbg/commands/ai.py [hunk 2]
```diff
@@ -364,6 +415,7 @@ def ai(question, model, temperature, max_tokens, verbose, list_models=False, com
     # print the arguments
     global last_question, last_answer, last_pc, last_command, verbosity
     ai_openai_api_key = get_openai_api_key()
+    ai_anthropic_api_key = get_anthropic_api_key()
```
Comment (disconnect3d) [inline pwndbg/commands/ai.py:417]: It seems that `ai_anthropic_api_key` variable is unused?

Comment (oblivia-simplex) [reply pwndbg/commands/ai.py:417]: ah, ok, fixed. i should be checking to see if *either* API key variable is set in the if statement that follows this.

File: pwndbg/commands/ai.py (hunk 1 above)
Comment (oblivia-simplex) [reply pwndbg/commands/ai.py]: yes. i just couldn't get textwrap to reliably make things pretty.

**Pull request merged**